CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## v0.6.1 (tbd)
   - allow copying from Depth32Float textures
```

Comment:
let's quote the actual type here i.e. `Depth32Float`

Reply:
Sorry, not sure what you mean? Just the backticks?